src/state/apis/zapper/zapperApi.ts

```diff
@@ -531,7 +556,9 @@ export const zapper = createApi({
                   })
 
                   const underlyingAssetId =
-                    asset.type === 'app-token' ? assetId : underlyingAssetIds[0]!
+                    asset.type === 'app-token' || asset.groupId === 'claimable'
```

Comment:
The guts of this PR - since we used the wrong underlying, it was being filtered out by `uniqBy` in `selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty` (e.g for Tokemak, using Tokemak as an underlyingAssetId instead of claimable TOKE means this was effectively a dupe of the TOKE opportunity).